apps/fz_common/lib/fz_map.ex

```diff
@@ -28,4 +28,24 @@ defmodule FzCommon.FzMap do
     end)
     |> Enum.into(%{})
   end
+
+  @doc """
+  Converts a map to a keyword list
+  """
+  def map_to_keyword_list(map) when is_map(map) do
+    map
+    |> Enum.into([], fn {k, v} ->
+      {
+        try do
+          String.to_existing_atom(k)
+        rescue
+          ArgumentError -> String.to_atom(k)
```

Comment:
This is the same as simply `String.to_atom` without `try`, isn't it?
Not that it's a recommendation. I'll add my recommendation to the config line.

Reply:
Yep, I misunderstood the docs and now realize that String.to_atom('foo') run multiple times will result in the same atom behind the scenes, similar to a Ruby symbol.